pkg/network/multus.go

```diff
@@ -17,6 +17,7 @@ func renderMultusConfig(manifestDir string) ([]*uns.Unstructured, error) {
 	data := render.MakeRenderData()
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["MultusImage"] = os.Getenv("MULTUS_IMAGE")
+	data.Data["SRIOVCNIImage"] = os.Getenv("SRIOV_CNI_IMAGE")
```

Comment:
Is this still necessary?

Reply:
Removed from ``renderMultusConfig`` and be rendered in ``renderOpenShiftSRIOV``.